builtin/provisioners/chef-client/ssh_provisioner.go

```diff
@@ -0,0 +1,72 @@
+package chefclient
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/hashicorp/terraform/communicator"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func (p *Provisioner) sshInstallChefClient(
+	o terraform.UIOutput,
+	comm communicator.Communicator) error {
+	var installCmd bytes.Buffer
+
+	// Build up a single command based on the given config options
+	installCmd.WriteString("curl")
```

Comment:
This is one reason why our [install.sh](https://www.chef.io/chef/install.sh) supports wget, and will fall back to python or perl to attempt to download.